navigation/styles2-navigation.css

```diff
@@ -0,0 +1,100 @@
+/* Navigation Main */
+.navigation-container {
+  margin: 3rem;
+}
+
+.navigation-main-heading {
+  color: yellow;
+}
+.navigation-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
+  height: 200px;
```

Comment:
use rems instead

Reply:
will do it next time..thanks for suggesting